frontend/sw360-portlet/src/main/webapp/html/components/includes/releases/editReleaseInformation.jspf

```diff
@@ -146,3 +146,16 @@
 
 	});
 </script>
+
+<core_rt:if test="${not empty autocompleteUrl && not empty autocompleteToken}">
```

Comment:
Why is `PortalConstants.CODESCOOP_URL` mapped to `autocompleteUrl` to be used here, instead of using `PortalConstants.CODESCOOP_URL` directly in some way here?

Reply:
@maxhbr general idea was that if some one wants do not destroy component data on portion update  you may use it like trigger for backend 
see https://github.com/eclipse/sw360/pull/300 where I provided fix
for now was decision  to escape from constant - will remove also from JSP

Reply:
This answer seems misplaced. I don't think it addresses @maxhbr's question.

Reply:
refactored